frontend/src/Utils/Api/APIService.tsx

```diff
@@ -11,7 +13,7 @@ export default class APIService {
       .catch((error) => console.error(`Login user error: ${error}`));
   }
 
-  static RegisterUser({ name, username, password, re_password, email }) {
+  static RegisterUser({ name, username, password, re_password, email }: RegisterProp) {
```

Comment:
`RegisterProp` => `RegisterUserConfiguration` 

`RegisterUser` is not an component, so it's arguments are not props